Add support for reporting an acceptable sentence. Fixes #83.

rolfbjarne opened the pull request:
Add a button to report that a sentence should not be flagged. For now this
just creates a new GitHub issue with the title and body filled in with the
sentence in question.

Fixes https://github.com/jonathanpeppers/inclusive-code-comments/issues/83.

jonathanpeppers reviewed (commented):
I like this, let's talk today if we need both buttons?

Maybe we just need a single "Report this sentence" button? So we wouldn't have that icon twice.

rolfbjarne commented:
I've changed the PR to modify the existing report button instead.

The created issue now looks like this (which is itself flagged because the issue contains the text that was flagged and is thus flagged again):

<img width="1274" alt="Screen Shot 2022-03-04 at 15 47 41" src="https://user-images.githubusercontent.com/249268/156784639-1173efbb-c53a-4594-a53c-e5100490eaba.png">

I also added a new label, `heuristics`, which is set for these issues, so we can easily filter them.

jonathanpeppers commented on src/content/errorCard.js:
```diff
@@ -161,6 +161,18 @@ class ErrorCard {
     _showGithubIssues(e) {
         e.stopImmediatePropagation();
         window.open("https://github.com/jonathanpeppers/inclusive-code-comments/issues/new");
+        let title = "Report sentence to improve heuristics";
+        let body = "The following word or sentence is acceptable:\n\n    " + this._error.originalPhrase + "\n\n";
+        body += "Additional info:\n\n";
+        body += "* Description: " + this._error.description + "\n";
+        body += "* Context phrase: " + this._error.contextPhrase + "\n";
+        body += "* isPartialValidation: " + this._error.isPartialValidation + "\n";
+        body += "* isPicky: " + this._error.isPicky + "\n";
+        body += "* isPunctuationError: " + this._error.isPunctuationError + "\n";
+        body += "* isSpellingError: " + this._error.isSpellingError + "\n";
+        body += "* isStyleError: " + this._error.isStyleError + "\n";
+        body += "* language: " + this._error.language.code + " (" + this._error.language.name + ")" + "\n";
+        window.open("https://github.com/jonathanpeppers/inclusive-code-comments/issues/new?labels=" + encodeURIComponent("enhancement,heuristics") + "&title=" + encodeURIComponent(title) + "&body=" + encodeURIComponent(body));
```
Maybe this was from a merge conflict: does this open two tabs on accident?
